modules/wallet/src/main/scala/org/tessellation/wallet/Main.scala

```diff
@@ -75,6 +78,12 @@ object Main
                     createCurrencyMessage[IO](keyPair, MessageType.Staking, address, parentOrdinal, outputPath)
                       .handleErrorWith(err => logger.error(err)(s"Error while creating staking signing message."))
                       .as(ExitCode.Success)
+                  case MergeSigningMessages(files, outputPath: Option[Path]) =>
+                    implicit val hasher = Hasher.forJson[IO]
+                    mergeMessages[IO](files)
+                      .flatMap(writeJson[IO, Signed[CurrencyMessage]](outputPath))
+                      .handleErrorWith(err => logger.error(err)(s"Error merging currency messages."))
+                      .as(ExitCode.Success)
```

Comment:
so in case of error, we end up with success exit code?

Reply:
@marcinwadon It's the same pattern used on all the commands. `ExitCode.Error` makes more sense to me for command-line tools though, which is the Unix convention.

Reply:
@marcinwadon @ryle-goehausen 
I could make this change to all the commands:
Do the `.as` so on success we exit with zero, but on error with one. WDYT?
```
.as(ExitCode.Success)
.handleErrorWith(err => logger.error(err)(s"Error merging currency messages.").as(ExitCode.Error))
```

Reply:
:+1: